scripts/buildDocs.ts

```diff
@@ -148,7 +148,7 @@ function buildSingleModuleDocumentation(documentation: IDocumentation, modulePat
     // tslint:disable-next-line:no-var-requires
     const module = require(modulePath);
     const DocumentedItem = module[documentation.exportName] as Documented;
-    if (DocumentedItem != null && DocumentedItem.metadata != null) {
+    if (DocumentedItem !== null && DocumentedItem.metadata !== null) {
```

Comment:
i'd prefer that it stay as `!=` instead of `!==`. `!= null` checks for both undefined and null whereas with `!==` we specify for null, which may not be the desired behavior. But happy to revisit if there's a good reason to change to `!==`!